misc/statistics.php

```diff
@@ -109,5 +110,6 @@ function format($str, $length)
     format($deviceTypes[8], $parsedUAs), getPercentage($deviceTypes[8], $parsedUAs),
     format($deviceTypes[9], $parsedUAs), getPercentage($deviceTypes[9], $parsedUAs),
     format($deviceTypes[10], $parsedUAs), getPercentage($deviceTypes[10], $parsedUAs),
+    format($deviceTypes[10], $parsedUAs), getPercentage($deviceTypes[13], $parsedUAs),
```

Comment:
Should this not be `format($deviceTypes[13], $parsedUAs), getPercentage($deviceTypes[13], $parsedUAs)`